### PR TITLE
Schemas

### PR DESCRIPTION
**Pull request by mhoc**

### CODE REVIEW

File: imports/api/Applications/index.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import { SimpleSchema } from 'meteor/aldeed:simple-schema'
+import { Meteor } from 'meteor/meteor'
+import { Mongo } from 'meteor/mongo'
+
+const ApplicationsSchema = new SimpleSchema({
+	text: {
+		type: String,
```
Comment (bwencke) [inline imports/api/Applications/index.js]: What is with this insane indentation?

Comment (garr741) [reply imports/api/Applications/index.js]: new computer, need help!

File: imports/api/Organizations/index.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+	description: {
+		type: String,
+	},
+	users: {
+		type: [String],	
+	},
+	positions: {
+		type: [String],
+	},
+	admins: {
```
Comment (bwencke) [inline imports/api/Organizations/index.js]: Remove

File: imports/api/Organizations/index.js [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+    type: String,
+  },
+  name: {
+    type: String,
+  },
+  description: {
+    type: String,
+  },
+  users: {
+    type: [String],
```
Comment (mhoc) [inline imports/api/Organizations/index.js:16]: Is this a denormalization of information in `Users.roles`?

**Pull request merged**